releases/client/2.3/settings

```diff
@@ -0,0 +1,3 @@
+FOREMAN_VERSION="2.3"
+FULLVERSION=$FOREMAN_VERSION
+OSES="rhel5 rhel6 rhel7 el8 sles11 sles12"
```

Comment:
a birdie told me, it's time to drop rhel5

Reply:
indeed

Reply:
updated to drop